Makefile

```diff
@@ -113,12 +118,26 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	make -C conmon clean
 	make -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+local-cross:
+	@$(MAKE) $(CROSS_BUILD_TARGETS); \
+	for file in $(CROSS_BUILD_TARGETS); do test -s $$file && exit 1 ; done
+
+.PHONY: bin/crio.cross.%
```

Comment:
Don't want to move this declaration down into the `.PHONY` block at the end of this `Makefile`?  I have no problem with declaring `.PHONY` either in one block or before each recipe, but I'd rather be consistent one way or the other.

Reply:
meh, i see the group of .PHONY at the bottom, but there are a couple that decorate the target they are phony'ing

Reply:
> meh, i see the group of `.PHONY` at the bottom, but there are a couple that decorate the target they are phony'ing

Fair enough.  I've filed #1594 to consolidate around the single block approach, but I'm fine addressing this instance as part of this PR or in follow-up work.